.buildkite/Dockerfile

```diff
@@ -1,13 +1,18 @@
-FROM gcr.io/opensourcecoin/radicle-registry/ci-base
+FROM gcr.io/opensourcecoin/radicle-registry/ci-base@sha256:18d8cf1e39f4f42d871a2a9d3bcd8641bfda343a1fc7b9fbca4f8aea422bae6f
 
 RUN apt-get -y update
 
 # install cypress deps
 RUN apt-get -y install autoconf git nettle-dev m4 gnupg xvfb libgtk-3-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2
 
+# install npm
```

Comment:
Why do we need this now?

Reply:
I couldn't get to the bottom of it, but without our `yarn install` step fails.